### ISSUE TITLE
[Pattern] Img alt enforcement

### ISSUE THREAD

**Issue by seanwuapps** [gov-components/gov-components#14]
**What is the pattern**
Screenshot or link to existing pattern/design

If an img tag doesn't have alt text, provide visual warning to the img tag.

**Purpose of this pattern**
Why should we implement this in our design system?

Enforce img alt text a11y compliance

**Composition**
What components should we use to create this pattern?

**Responsiveness**
Specify any behaviour change between Mobile -> Tablet -> Desktop

**User interactions**
What happens when user click/focus/hover/touch it? Any external state to be considered? (e.g. if the user is logged in)
